boa3/builtin/compile_time/__init__.py

```diff
@@ -306,7 +306,8 @@ def add_group(self, pub_key: str, signature: str):
         if new_group not in self._permissions:
             self._groups.append(new_group)
 
-    def add_permission(self, *, contract: str = IMPORT_WILDCARD, methods: list[str, str] = IMPORT_WILDCARD):
+    def add_permission(self, *, contract: str = IMPORT_WILDCARD,
+                       methods: list[str] | str | tuple = IMPORT_WILDCARD):
```

Comment:
just an observation for tuple type annotation: you can annotate a tuple of any size like this: `tuple[str, ...]`